lib/cfn-nag/custom_rules/CodeBuildEncryptionKeyRule.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+# Rule class to ensure a CF distribution has logging
+class CodeBuildEncryptionKeyRule < BaseRule
+  def rule_text
+    'CodeBuild project should specify an EncryptionKey value'
+  end
+
+  def rule_type
+    Violation::WARNING
```

Comment:
hmm... don't recall what the issue asks for but we usually treat missing encryption settings as a fail.  on the other hand, the default without a key is to use the amazon cmk which might be ok for some folks